HousingRepairsOnline.AddressIngestion/Helpers/Constants.cs

```diff
@@ -0,0 +1,6 @@
+namespace HousingRepairsOnline.AddressIngestion.Helpers;
+
+public static class Constants
+{
+    public const string HousingProvider = "Capita";
```

Comment:
Ideally, this would be a configuration value passed in when the function is invoked.
This would keep the codebase agnostic of any specific system.

Is it possible to provide the Housing Provide when the function is invoked? Or equivalent?